### PR TITLE
[GpxRouteApproximation] Sync finalPoints.routeToTarget with final route result

### PR DESCRIPTION
**Pull request by RZR-UA**

### CODE REVIEW

**Comment by RZR-UA** [comment]
https://github.com/osmandapp/OsmAnd-Issues/issues/2342

**Comment by RZR-UA** [comment]
server: https://github.com/osmandapp/OsmAnd-tools/pull/849

File: OsmAnd-java/src/main/java/net/osmand/router/RoutePlannerFrontEnd.java [hunk 1]
```diff
@@ -80,25 +74,24 @@ public void setHHRouteCpp(boolean cpp) {
 	
 	public static class GpxRouteApproximation {
 		// ! MAIN parameter to approximate (35m good for custom recorded tracks) 
-		public double MINIMUM_POINT_APPROXIMATION = 200; // 35 m good for small deviations
+		// public double MINIMUM_POINT_APPROXIMATION = 200; // 35 m good for small deviations
```
Comment (vshcherb) [inline OsmAnd-java/src/main/java/net/osmand/router/RoutePlannerFrontEnd.java:77]: We need to delete this block

Comment (RZR-UA) [reply OsmAnd-java/src/main/java/net/osmand/router/RoutePlannerFrontEnd.java]: cleaned up

**Pull request merged**